### ISSUE TITLE
Ability to personnalize Wallpaper in the App

### ISSUE THREAD

**Issue by CBGTI** [rocksdanister/lively#1273]
I just see tha I lost the possibility to personnalize my Wallpaper (constrast, saturation, tint...).
The option is grayed for all.
I don't know since how many time. 

![Sans titre](https://user-images.githubusercontent.com/89838333/175290907-f10a1962-a11b-4e4b-a467-999c8538f230.png)

**Comment by rocksdanister** [comment]
That's odd, can you set one of the wallpaper and then share the logfile:
Goto Lively settings (Gear icon) -> Misc. -> Log File

**Comment by CBGTI** [comment]
@rocksdanister :  Here's my Log.  

[lively_log_20220625_211510.zip](https://github.com/rocksdanister/lively/files/8985527/lively_log_20220625_211510.zip)

**Comment by rocksdanister** [comment]
Check if this file is present:
`%LocalAppData%\Packages\12030rocksdanister.LivelyWallpaper_97hta09mmv6hy\LocalCache\Local\Lively Wallpaper\Mpv\LivelyProperties.json`

**Comment by CBGTI** [comment]
I have this file, but not exactly at the same place :  
E:\WindowsApps\12030rocksdanister.LivelyWallpaper_1.0.122.0_x86__97hta09mmv6hy\Build\Plugins\Mpv\api\LivelyProperties.json

**Comment by rocksdanister** [comment]
How did you change the application install location for Windowsapps? somethings off.

**Comment by CBGTI** [comment]
Yes. But No folder change since the first installation of Windows.
It's my real location for all my App from the Windows Store.

**Comment by rocksdanister** [comment]
Open this folder by clicking the marked button and share the location address:
<img width="480" alt="image" src="https://user-images.githubusercontent.com/17554161/175823985-33c1cd3a-1aec-47d4-8cc7-066fa16c97b7.png">

Example:
`C:\Users\rocks\AppData\Local\Lively Wallpaper\Library`

**Comment by CBGTI** [comment]
Here's mine.

![Sans titre](https://user-images.githubusercontent.com/89838333/175824829-2a9f124b-96f8-4a43-852d-af9eb039feab.jpg)

**Comment by rocksdanister** [comment]
Just copy the `LivelyProperties.json` file here:
`%LocalAppData%\Packages\12030rocksdanister.LivelyWallpaper_97hta09mmv6hy\LocalCache\Local\Lively Wallpaper\Mpv\LivelyProperties.json`

and restart the app.

**Comment by CBGTI** [comment]
I put a copy of the "LivelyProperties.json" in the place your told me, but just after restarting your App, the file is automatically deleted from this folder.  It's very strange but I tried 3 times with the same result.     O_o
